java/spi.java.hints/test/unit/src/org/netbeans/modules/java/hints/spiimpl/UtilitiesTest.java

```diff
@@ -380,9 +380,9 @@ public void testCasePattern() throws Exception {
         String golden = "case $expr: foo bar; $stmts$; ";
         assertEquals(golden.replaceAll("[ \n\r]+", " "), result.toString().replaceAll("[ \n\r]+", " "));
         assertDiagnostics(errors, "19-19:compiler.err.expected");
-        assertPositions(result, positions[0], code, "$expr", "$stmts$", "$stmts$;", "case $expr: foo bar $stmts$;", "foo", "foo bar ");
+        assertPositions(result, positions[0], code, "$expr", "$expr", "$stmts$", "$stmts$;", "case $expr: foo bar $stmts$;", "foo", "foo bar ");
     }
-*/
```

Comment:
awesome. Good to see this test case enabled.
I wanted to do that in:
https://github.com/apache/netbeans/pull/5107#discussion_r1047673049
when we enabled tests for this module (and many others) which was completely turned off till now.

but I wasn't sure if the result was actually correct. So I commented it out instead of running the risk to commit a wrong test assumption.